kernel/src/kcb.rs

```diff
@@ -355,6 +364,10 @@ impl<A: ArchSpecificKcb> Kcb<A> {
         self.physical_memory.gmanager = Some(gm);
```

Comment:
rename to `set_global_mem` for naming consistency (persistent memory -> pmem, volatile memory -> mem)

Reply:
Updated.